python/popgetter/metadata.py

```diff
@@ -10,6 +11,64 @@
 from pydantic import BaseModel, Field, computed_field, model_validator
 
 
+class COL(Enum):
+    """
+    This class stores the eventual column names of all the metadata classes,
+    which are used when serialising the metadata to a dataframe.
+    """
+
+    GEO_ID = "GEO_ID"
+
+    COUNTRY_ID = "country_id"
+    COUNTRY_NAME_SHORT_EN = "country_name_short_en"
+    COUNTRY_NAME_OFFICIAL = "country_name_official"
+    COUNTRY_ISO3 = "country_iso3"
+    COUNTRY_ISO2 = "country_iso2"
+    COUNTRY_ISO3166_2 = "country_iso3166_2"
+
+    PUBLISHER_ID = "data_publisher_id"
+    PUBLISHER_NAME = "data_publisher_name"
+    PUBLISHER_URL = "data_publisher_url"
+    PUBLISHER_DESCRIPTION = "data_publisher_description"
+    PUBLISHER_COUNTRIES_OF_INTEREST = "data_publisher_countries_of_interest"
```

Comment:
Apologies, for completeness, perhaps this last one too!
```suggestion
    DATA_PUBLISHER_ID = "data_publisher_id"
    DATA_PUBLISHER_NAME = "data_publisher_name"
    DATA_PUBLISHER_URL = "data_publisher_url"
    DATA_PUBLISHER_DESCRIPTION = "data_publisher_description"
    DATA_PUBLISHER_COUNTRIES_OF_INTEREST = "data_publisher_countries_of_interest"
```

Reply:
Oh, I was kind of torn for this one because I felt like if I did DATA_PUBLISHER_... I'd also have to do SOURCE_DATA_RELEASE_... haha

Reply:
(Should we just do the full names?)

Reply:
Oh yeah, good point! I suppose the downside would be if it is used widely in the code and makes the code less readable.

Reply:
It looks like these ones will just be in the metadata? In that case, perhaps naming with full name for both is best for consistency.